src/components/progress/progressRing.svelte

```diff
@@ -17,6 +17,7 @@
       style:cx="var(--radius)"
       style:cy="var(--radius)"
       style:stroke-width="var(--stroke-width)"
+      style:filter="opacity(0.3)"
```

Comment:
Hahaha was talking about whether it was left in accidentally :D But yeah, might as well set in the the CSS, rather than with a style tag :)

Reply:
Given that we already have a `style` attribute, do you think there's a particular advantage in doing so? Only asking because we don't currently have a unique selector to use for the background circle.

Reply:
Considering we already have some css for the circle, it would be nice to move these styles to the css so the style rules aren't continuing to be split?